app/observers/live_events_observer.rb

```diff
@@ -19,9 +20,17 @@ def after_update(obj)
       if obj.title_changed? || obj.body_changed?
         Canvas::LiveEvents.wiki_page_updated(obj, obj.title_changed? ? obj.title_was : nil,
                                                   obj.body_changed? ? obj.body_was : nil)
+      elsif obj.workflow_state_changed? && obj.workflow_state == 'deleted'
+        Canvas::LiveEvents.wiki_page_deleted(obj)
       end
     when Assignment
       Canvas::LiveEvents.assignment_updated(obj)
+    when Attachment
+      if obj.display_name_changed?
+        Canvas::LiveEvents.attachment_updated(obj, obj.display_name_was)
+      elsif obj.file_state_changed? && obj.file_state == 'deleted'
+        Canvas::LiveEvents.attachment_deleted(obj)
+      end
```

Comment:
Possibly, but they seemed to have narrowed wiki pages to title/body as well. We could inverse this to something like:

```ruby
if obj.file_state_changed? && obj.file_state == 'deleted'
   Canvas::LiveEvents.attachment_deleted(obj)
elsif obj.changed?
   Canvas::LiveEvents.attachment_updated(obj)
end
```

which would cover pretty much anything, but I wanted to keep it inline with the wiki pages.